pkg/files/file.go

```diff
@@ -52,7 +52,7 @@ func NewSortedFilesFromPaths(paths []string) ([]*File, error) {
 		default:
 			fileInfo, err := os.Lstat(path)
 			if err != nil {
-				return nil, fmt.Errorf("Checking file '%s'", path)
+				return nil, fmt.Errorf("Checking file '%s', %v", path, err)
```

Comment:
good catch. let's change to `fmt.Errorf("Checking file '%s': %s", path, err)` to be more consistent with other error wraps